lib/manager/helmv3/artifacts.spec.ts

```diff
@@ -1,24 +1,17 @@
-import { exec as _exec } from 'child_process';
-import _fs from 'fs-extra';
 import { join } from 'upath';
-import { envMock, mockExecAll } from '../../../test/exec-util';
-import { loadFixture, mocked } from '../../../test/util';
+import { envMock, git, exec, mockExecAll } from '../../../test/exec-util';
+import { env, fs, loadFixture, mocked } from '../../../test/util';
 import { GlobalConfig } from '../../config/global';
 import type { RepoGlobalConfig } from '../../config/types';
 import * as docker from '../../util/exec/docker';
-import * as _env from '../../util/exec/env';
 import * as hostRules from '../../util/host-rules';
 import type { UpdateArtifactsConfig } from '../types';
 import * as helmv3 from './artifacts';
 
-jest.mock('fs-extra');
 jest.mock('child_process');
 jest.mock('../../util/exec/env');
 jest.mock('../../util/http');
-
-const fs: jest.Mocked<typeof _fs> = _fs as any;
-const exec: jest.Mock<typeof _exec> = _exec as any;
-const env = mocked(_env);
+jest.mock('../../util/fs');
```

Comment:
I think we should use our new Fixtures class to mock files, can you check other managers for samples?